app/src/main/java/com/example/series_collector/data/room/AppDatabase.kt

```diff
@@ -10,9 +11,9 @@ import androidx.work.OneTimeWorkRequestBuilder
 import androidx.work.WorkManager
 import androidx.work.workDataOf
 import com.example.series_collector.data.Series
-import com.example.series_collector.utils.workers.SeriesDatabaseWorker
+import com.example.series_collector.utils.workers.SeriesInitWorker
 
-@Database(entities = [Series::class], version = 1, exportSchema = false)
+@Database(entities = [Series::class], version = 2, exportSchema = false)
 @TypeConverters(Converter::class)
 abstract class AppDatabase : RoomDatabase() {
     abstract fun SeriesDao(): SeriesDao
```

Comment:
```suggestion
    abstract fun seriesDao(): SeriesDao
```
https://kotlinlang.org/docs/coding-conventions.html